hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -44,15 +44,10 @@ The protocol controller currently supports the following features:
 *  Flash memory protection at page boundaries.
 *  Life cycle RMA entry.
 *  Key manager secret seeds that are inaccessible to software.
-*  Features to be added if required
-   *  Program verification
-      *  may not be required if flash memory supports alternative mechanisms of verification.
-   *  Erase verification
-      *  may not be required if flash memory supports alternative mechanisms of verification.
-   *  Flash redundant pages
-      *  Flash may contain additional pages used to remap broken pages for yield recovery.
-      *  The storage, loading and security of redundant pages may also be implemented in the physical controller or flash memory.
-
+*  Erase suspend to vendor flash module.
```

Comment:
This sentence is confusing: I assume it sends some info to the vendor, does it send an "erase suspension"? Ah... please add a link to the Erase suspend section.

Reply:
fair point let me make this an actual sentence hehe.